tiny_dnn/layers/average_pooling_layer.h

```diff
@@ -41,8 +41,9 @@ inline void tiny_average_pooling_kernel(
       float_t bias   = b.host_at(d);
       for (size_t i = 0; i < oarea; ++i, ++idx) {
         const auto &connections = out2wi[idx];
-        float_t value = 0.0;
-        for (auto connection : connections) value += in.host_at(connection.second);
+        float_t value           = 0.0;
+        for (auto connection : connections)
+          value += in.host_at(connection.second);
```

Comment:
@Randl can we just have `in.at(...)` and let tensor decide whether use host or device ?